package.json

```diff
@@ -10,12 +10,16 @@
     "packages/*",
     "build-settings"
   ],
+  "engines": {
+    "node": ">=16"
+  },
```

Comment:
I don't think we should include this - this will only cause things to break for folks who aren't upgraded yet with no upside. I think it's ok to not specify the engine!

Reply:
I'll change it to 12. This only affects folks developing and building Wonder Stuff, so I think it's still worth having to prevent working in the wrong version entirely.